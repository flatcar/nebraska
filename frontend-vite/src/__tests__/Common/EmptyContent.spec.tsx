import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import Empty from '../../components/common/EmptyContent';

describe('Empty', () => {
  it('renders correct content', () => {
    const { asFragment } = render(<Empty>Some Dummy Content</Empty>);
    expect(asFragment()).toMatchSnapshot();
  });
});
