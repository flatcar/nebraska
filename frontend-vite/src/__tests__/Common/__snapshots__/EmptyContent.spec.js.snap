// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty renders correct content 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-19midj6"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter css-12wjmzy-MuiTypography-root"
      data-testid="empty"
    >
      Some Dummy Content
    </p>
  </div>
</DocumentFragment>
`;
